crud/src/main/java/org/openmbee/mms/crud/services/NodeOperation.java

```diff
@@ -130,7 +131,7 @@ public void processElementAdded(ElementJson e, Node n, NodeChangeInfo info) {
         cmjs.getAdded().add(newObj);
 
         n.setNodeId(e.getId());
-        n.setInitialCommit(e.getDocId());
+        n.setInitialCommit(e.getCommitId());
```

Comment:
this was actually a misnomer and the initial commit was referring to the initial element's doc id - i don't think that's relevant anymore though, just fyi (i think it was because of the initial commit object issue before too)